components/dashboard/src/projects/NewProject.tsx

```diff
@@ -300,15 +276,24 @@ export default function NewProject() {
         const showSearchInput = !!repoSearchFilter || filteredRepos.length > 0;
 
         const renderRepos = () => (<>
-            {!isBitbucket() && <p className="text-gray-500 text-center text-base">Select a Git repository on <strong>{selectedProviderHost}</strong>. (<a className="gp-link cursor-pointer" onClick={() => setShowGitProviders(true)}>change</a>)</p>}
-            <div className={`mt-10 border rounded-xl border-gray-100 dark:border-gray-800 flex-col`}>
-                <div className="px-8 pt-8 flex flex-col space-y-2" data-analytics='{"label":"Identity"}'>
+            <p className="text-gray-500 text-center text-base">Projects allow you to manage prebuilds and workspaces for your repository. <a href="https://www.gitpod.io/docs/teams-and-projects" rel="noopener" className="gp-link">Learn more</a></p>
+            <p className="text-gray-500 text-center text-base mt-12">Select account on <b>{selectedProviderHost}</b> (<a className="gp-link cursor-pointer" onClick={() => setShowGitProviders(true)}>change</a>)</p>
+            <div className={`mt-2 flex-col w-96`}>
+                <div className="px-8 pt-5 flex flex-col space-y-2" data-analytics='{"label":"Identity"}'>
                     <ContextMenu classes="w-full left-0 cursor-pointer" menuEntries={getDropDownEntries(accounts)}>
                         <div className="w-full">
-                            {icon && (
-                                <img src={icon} className="rounded-full w-6 h-6 absolute top-1/4 left-4" />
+                            {!selectedAccount && user && user.name && user.avatarUrl && (
+                                <>
+                                    <img src={user?.avatarUrl} className="rounded-full w-6 h-6 absolute top-1/4 left-3" />
```

Comment:
> did we drop the idea of the reconfigure link at the bottom from [#7163 (comment)](https://github.com/gitpod-io/gitpod/pull/7163#issuecomment-990914397)?

Thanks for redirecting me to this part that I missed. Will put it back.

Reply:
> Once we change the dropdown height, using `top-2` instead of `top-1/4` could suffice. However, maybe this is something we could also tackle in a next step as it's a minor issue which was already existing before this PR.

@gtsiolis Ah, I see. Would you mind pointing me to where this 42px is set? If it's a straightforward change then I'll do it here, otherwise we can tackle it later.

Reply:
@laushinka I don't think the `42px` height is directly set anywhere as it's because of the padding used from the inner elements of the dropdown.

<img width="474" alt="Screenshot 2021-12-14 at 8 43 58 PM" src="https://user-images.githubusercontent.com/120486/146060335-6be9a91a-88fa-48a5-aea9-05cddbb609c4.png">

Reply:
Did a little hack which pushed up the avatar a tiny bit, but still not exactly centered. I'll try to play a bit before this PR closes, but if not then in the next issue 🙏🏽

Reply:
> I'll try to play a bit before this PR closes, but if not then in the next issue

DEAL 🤝